### DISCUSSION TITLE
OpenSABER Building Blocks

### DISCUSSION THREAD

**Discussion by surendrasinghs** [Sunbird-RC/community#6] [General]
Can you help me understand various building blocks of registry - OpenSABER? 

It would be great if you can share 
1. Are there any user facing apps which are leveraging the micro services to deliver a self-service experience?
2. List of micro services 
3. Any pluggable tool or widgets?

@pramodkvarma @dileepbapat @vrayulu

**Comment by pramodkvarma** [comment]
Can you explain the purpose a bit more @surendrasinghs ? What exactly are you trying to achieve?
